repair_scheduled_date_calendar_view/readme/DESCRIPTION.md

```
@@ -0,0 +1,3 @@
+This module implements a calendar view for repair orders. The view Is based on the scheduled date. If you need to use calendar view based on the date when the order was done consider using the repair_calendar_view module instead.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)